### ISSUE TITLE
“${jdbc.datasource.default}” Does not resolve

### ISSUE THREAD

**Issue by Laurens-makel** [ibissource/iaf#648]
**Describe the bug**
In #547 the following was introduced:

![image](https://user-images.githubusercontent.com/37303445/80470691-a4907980-8942-11ea-8380-8a1af1d3979d.png)

It seems that in the JdbcFacade class the value for proxiedDataSources which contains “${jdbc.datasource.default}” property does not resolve to the default value.

**Category**
 - Frank Framework
 - Larva (unit test tool)

**To Reproduce**

This is a little difficult because it's a customer specific testing setup that includes Larva tests as JUnit tests to run them during build time.

If specified files are needed to debug, feel drop a comment and will provide them privately.

**Expected behavior**
“${jdbc.datasource.default}” resolves to it's default value.

**Screenshots**
![image](https://user-images.githubusercontent.com/37303445/80473517-8dec2180-8946-11ea-9520-6d57b54ae1ca.png)

![image](https://user-images.githubusercontent.com/37303445/80473454-74e37080-8946-11ea-9b72-05aa4aa3fbeb.png)

**Environment:**
 - Application server (f.e. Tomcat/websphere/JBOSS/Jetty): IBISTEST
 - Browser [e.g. chrome, safari]: None
 - IAF Version [e.g. 7.4]: 7.6-20200330.192854

**Additional context**

```
2020-04-25 16:30:26,921 WARN  [main] null jdbc.XmlQuerySender - [nl.nn.adapterframework.jdbc.XmlQuerySender] [Query-sender] exception retrieving PhysicalDestinationName
nl.nn.adapterframework.jdbc.JdbcException: Could not find Datasource [jdbc/ibis4titan]: (NoInitialContextException) Need to specify class name in environment or system property, or as an applet parameter, or in an application resource file:  java.naming.factory.initial
               at nl.nn.adapterframework.jdbc.JdbcFacade.getDatasource(JdbcFacade.java:123)
               at nl.nn.adapterframework.jdbc.JdbcFacade.getConnection(JdbcFacade.java:236)
               at nl.nn.adapterframework.jdbc.JdbcFacade.getPhysicalDestinationName(JdbcFacade.java:270)
               at nl.nn.adapterframework.pipes.MessageSendingPipe.configure(MessageSendingPipe.java:300)

```

**Comment by nielsm5** [comment]
It looks like the value is resolved as expected. Are you using a XmlQuerySender with the datasourceName attribute, or are you using the jmsRealm attribute?

If you're using jmsRealm, make sure you're not overwriting the value in another (sub-) configuration.

**Comment by rvdstel** [comment]
This error comes from the IAF adapter **ManageDatabase**, it cannot find the datasource.

This adapter is changed in the new IAF version,  :
from : jmsRealm="jdbc" 
to : datasourceName="${jdbc.datasource.default}"

What should we specify where to make this work again? 

We only specify this : 
**Configuration.xml:** 
	<jmsRealms>
		<jmsRealm datasourceName="jdbc/ibis4titan" realmName="jdbc"/>
	</jmsRealms>

**DeploymentSpecifics.properties:**
jdbc.migrator.active=true
jdbc.migrator.dataSource=jdbc/ibis4titan

**Comment by nielsm5** [comment]
Hi,

I cannot reproduce this, do you guys have any custom code?

**Comment by rvdstel** [comment]
There is no custom code.
You can reproduce by : 
1. cloning one of our projects: https://vcs.aws.insim.biz/ibissource/dm/ibis4titan/ibis4titan_titan
2. Change the IAF version in the pom.xml to : 7.6-20200317.120259 or higher
3. Run the Junit LarvaTest class in src\test\java\test
4. ERRORS: adapter [ManageDatabase] has state [**ERROR**], will retry...

with version 7.6-20200317.120259 all tests will pass.

**Comment by jjansenvr** [comment]
@Laurens-makel is this fixed?

**Comment by nielsm5** [comment]
I get te same Liquibase errors when I specify the datasource manually. When I don't set the property it works...

**Comment by nielsm5** [comment]
Might be fixed in #1415
